src/utils/wallet.js

```diff
@@ -440,7 +440,9 @@ class Wallet {
         const accountsIds = await getAccountId(publicKey)
 
         if (!accountsIds.length) {
-            throw new Error(`Cannot find matching public key`);
+            const error = new Error(`Cannot find matching public key`)
```

Comment:
I think it makes sense to define some custom type like `WalletError`, so that you can do it like:

```
throw new WalletError('account.recoverAccount.errorInvalidSeedPhrase', { publicKey });
```

and it would look up message automatically (and also include relevant parameters like `publicKey`)